src/main/kotlin/gg/essential/essential-loader.gradle.kts

```diff
@@ -0,0 +1,68 @@
+package gg.essential
+
+import gg.essential.gradle.multiversion.Platform
+import gg.essential.gradle.util.RelocationTransform.Companion.registerRelocationAttribute
+
+
+plugins {
+    id("gg.essential.loom")
+}
+
+val platform = Platform.of(project)
+
+val essentialLoader: Configuration by configurations.creating
+
+when {
+    platform.isLegacyForge -> {
+        dependencies {
+            "implementation"(essentialLoader("gg.essential:loader-launchwrapper:1.1.3")!!)
+        }
+        tasks.named<Jar>("jar") {
+            dependsOn(essentialLoader)
+            manifest.attributes(mapOf(
+                "FMLCorePluginContainsMod" to "Yes, yes it does",
```

Comment:
What's this for? Afaict it only [applies to coremods](https://github.com/MinecraftForge/MinecraftForge/blob/1.12.x/src/main/java/net/minecraftforge/fml/relauncher/CoreModManager.java#L434) and [isn't even checked for tweakers](https://github.com/MinecraftForge/MinecraftForge/blob/1.12.x/src/main/java/net/minecraftforge/fml/relauncher/CoreModManager.java#L395-L404).